src/templates/healthcert/types.tsx

```diff
@@ -1,6 +1,12 @@
 import { v2 } from "@govtechsg/decentralized-renderer-react-components";
 import { pdtHealthcert as healthcert, notarise } from "@govtechsg/oa-schemata";
+import { PDTHealthCertV2 } from "./tempTypes";
```

Comment:
should rename the filename something like `v2Types`. `temp` seem like temporary , i think you name it for template.

Reply:
Yes, it's a temporary file because https://github.com/Open-Attestation/schemata/pull/38 not published yet.

Reply:
imm... but it's not temporary. will use it after v1 is fully deprecated right ?

Reply:
Temporary because we will import the `PDTHealthCertV2` type directly from `@govtechsg/oa-schemata` when it eventually gets published. For example:

```javascript
import { PDTHealthCertV2 } from "@govtechsg/oa-schemata";
```